{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "psirt@us.ibm.com",
      "ID": "CVE-2020-4901"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Robotic Process Automation with Automation Anywhere 11.0 could allow an attacker on the network to obtain sensitive information or cause a denial of service through username enumeration. IBM X-Force ID: 190992."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://www.ibm.com/support/pages/node/6450435",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://www.ibm.com/support/pages/node/6450435"
        },
        {
          "name": "ibm-rpa-cve20204901-info-disc (190992)",
          "refsource": "XF",
          "tags": [],
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/190992"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-05-07T16:15Z",
  "publishedDate": "2021-05-07T16:15Z"
}